uint8 detection_success
float32 detection_confidence
float32 time_stamp 

clm_ros_wrapper/ClmHeadPose headpose
clm_ros_wrapper/ClmEyeGaze[] eyegazes
clm_ros_wrapper/ClmFacialActionUnit[] aus
